phy/cluster/manual/clustering.py

```diff
@@ -6,15 +6,14 @@
 # Imports
 #------------------------------------------------------------------------------
 
-from collections import namedtuple, defaultdict, OrderedDict
-from copy import deepcopy
-
 import numpy as np
 
-from ...ext.six import iterkeys, itervalues, iteritems
 from ...utils.array import _as_array, _is_array_like
```

Comment:
from ...something import foo
is better written:
from phy.something import foo

easier to understand, to grep, to refactor.
only from .something import blabla make sense to me. (because it's a local import in the same module)